owncloudApp/src/main/java/com/owncloud/android/presentation/files/createfolder/CreateFolderDialogFragment.kt

```diff
@@ -43,6 +46,8 @@ class CreateFolderDialogFragment : DialogFragment() {
     private lateinit var parentFolder: OCFile
     private lateinit var createFolderListener: CreateFolderListener
     private var isButtonEnabled: Boolean = false
+    private val MAX_FILENAME_LENGTH = 223
+    private val pattern = "[/\\\\]".toRegex()
```

Comment:
I think here should be enough with the `[/\\]` expression for the regular expression. The `[]` indicate that the String matches the regex if it contains one of the characters inside them, and in this case you are repeating the `\` character (with a total of 4 `\` because you need to escape this character)

Reply:
We need 4 \ for creating the regular expression in a good way. There are more details in this link: https://stackoverflow.com/a/57477268